### DISCUSSION TITLE
Question on how to loop mods

### DISCUSSION THREAD

**Discussion by Maxi2022gt** [XeroOl/notitg-mirin#92] [General]
How do i loop a mod like this:
Apply skewx with 100%
After 1 beat, Apply skewx but this time with -100%

**Comment by silvashp** [comment]
for beat = starting beat, end beat, 2 do
ease{beat, easelength, easeoption, 100, 'skewx'
ease{beat+1, easelength, easeoption, -100, 'skewx'
end

**Reply by Maxi2022gt** [reply]
doesn't work

**Reply by Maxi2022gt** [reply]
oh it worked
i just had to edit the code a bit
also the fixed code is
`for beat = starting, ending, 2 do`
`	ease {beat, 1, outExpo, 100, 'skewx'}`
`	ease {beat+1, 1, outExpo, -100, 'skewx'}`
`end`
